include/swift/AST/Module.h

```diff
@@ -638,6 +656,13 @@ class FileUnit : public DeclContext {
     return getImportedModules(imports, ModuleDecl::ImportFilter::Public);
   }
 
+  /// \see ModuleDecl::getImportedModulesForLinking
+  virtual void getImportedModulesForLinking(
+      SmallVectorImpl<ModuleDecl::ImportedModule> &imports) const {
+    return getImportedModules(imports, ModuleDecl::ImportFilter::Public);
+    return getImportedModules(imports, ModuleDecl::ImportFilter::UsableFromInline);
```

Comment:
Copy/paste error here. Should there have been a test that catches this, or is it just that this is overridden everywhere in practice?

Reply:
I'm not sure what the error is -- the set of modules for linking includes public (@_exported) imports, and @_usableFromInline imports.

Reply:
There's a `return`, so the second line never happens.

Reply:
Oops, thanks. Yeah, I'll add a test. You need to have both an @_exported and @_usableFromInline import.